### DISCUSSION TITLE
Target 29/11/2023 - Don't know what to do…

### DISCUSSION THREAD

**Discussion by Cinquantesix** [cssbattle/forum#369] [Bugs reports]
Hello,

I tried to reach the 100% but I don't know what I'm supposed to do… I just take a screenshot because, I think I cannot do something more… When I switch to "diff" mode, I just can't see anywhere is the problem… I tried different solutions but none seem to work.

What can I do?
![Capture d’écran 2023-11-29 à 09 44 07](https://github.com/cssbattle/forum/assets/23057213/44481b35-5f42-47b8-9f77-510416a23e5a)

**Comment by PavButsa** [comment]
Hello! By now, 105 players have succeeded in today's daily target. I can give you a hint (not a full solution) if it is not forbidden here.
@chinchang , may I publish my idea of solution here?

**Comment by chinchang** [comment]
Yeah, i guess just a hint is fine

**Comment by PavButsa** [comment]
I used two gradients (radial and conic) for background of the html element to make a circle part. Then I placed an element with border-right and border-bottom rotated on 45 degrees to make a "V" part. Hope it will help you!

**Reply by Cinquantesix** [reply]
Thanks, I'll try this way!
